src/components/file-upload/index.md

```diff
@@ -66,4 +66,10 @@ Say ‘The selected file must use the template’.
 
 ## Research on this component
 
+An accessibility audit has shown that some users struggle with file upload.
+
+### Known issues and gaps
+
+The file upload component has no visual target area when dragging and dropping a file. This usability issue is due to the fact that our component inherits and uses browser default behaviour. We don’t plan on fixing this at the moment, as it’s determined by the user agent, though we will take it into consideration if we ever publish a more custom file upload component. For more detail and to track our progress we have the [GitHub issue: ‘Upload file component has no visual target area when dragging and dropping a file’](https://github.com/alphagov/govuk-frontend/issues/3685).
```

Comment:
Not a requirement, but with my content cap on, it feels like the wording could be tightened up a little.

```suggestion
The file upload component doesn't show a visual target area when dragging and dropping a file. We don’t plan on fixing this at the moment, as the component inherits and uses the browser's default behaviour, though we will take it into consideration if we ever publish a custom file upload component. For more detail and to track our progress we have the [GitHub issue: ‘Upload file component has no visual target area when dragging and dropping a file’](https://github.com/alphagov/govuk-frontend/issues/3685).
```